python/lancedb/db.py

```diff
@@ -192,6 +192,13 @@ def drop_table(self, name: str):
         """
         raise NotImplementedError
 
+    def drop_database(self):
+        """
+        Drop all the tables at once
+        or delete the database at once
```

Comment:
```suggestion
        """Drop all tables
```

The phrasing with the `or` is a little confusing to me.  I think you are trying to say "this does X or Y since both X and Y are really the same idea"

However, I read this as "this can do X or Y" and I am left wondering how to figure out what flag controls this (i.e. I'm looking for a "operation_type" enum or something)

If it's important to communicate that these are the same thing you could do something like...

```
"""Delete the database

This is the same thing as dropping all of the tables
```

The "at once" phrasing is redundant I think and a user might wonder if it means that the operation is atomic / thread safe.  This operation is not "atomic". If a user calls `table_names` while a `drop_database` call is in progress then they might see a partial result.